pubspec.yaml

```diff
@@ -66,13 +66,23 @@ dependencies:
   # Allow using feather icons (nice outlined icons)
   flutter_feather_icons: ^1.0.2
 
+
   # Package that allows for supporting multiple languages
   flutter_localizations:
     sdk: flutter
 
   # Display markdown-formatted text
   flutter_markdown: ^0.4.4
 
+  # Use an in-app browser to ask user for Google Calendar permission
+  flutter_web_browser: any
```

Comment:
Don't use any, use the version you actually used in the implementation. Otherwise if the package has a breaking change, our code will break too.